### ISSUE TITLE
CSCwi58743 - Syntax issue in CISCO-RTTMON-TC-MIB.my

### ISSUE THREAD

**Issue by kyzer-davis** [cisco/cisco-mibs#53]
See #47

**Comment by Hipska** [comment]
At least 2 external contributors already proposed a fix for this.

**Comment by kyzer-davis** [comment]
Yeah, #47, #109 and now #130. Unfortunately I can't accept these PRs until the internal file is fixed.

See: https://github.com/cisco/cisco-mibs/blob/main/CONTRIBUTING.md#repeat-pull-requests

**Comment by Hipska** [comment]
What is the status of `CSCwi58743`? It looks like only 5 min of work..
